src/activity_actor.cpp

```diff
@@ -4010,6 +4010,14 @@ std::unique_ptr<activity_actor> insert_item_activity_actor::deserialize( JsonVal
     return actor.clone();
 }
 
+reload_activity_actor::reload_activity_actor( item::reload_option &&opt, int extra_moves )
+{
+    moves_total = opt.moves() + extra_moves;
+    quantity = opt.qty();
+    reload_targets.emplace_back( std::move( opt.target ) );
+    reload_targets.emplace_back( std::move( opt.ammo ) );
```

Comment:
Would be great if you also got rid of the `reload_targets` vector, that's probably just a holdover from the old activity system.

Reply:
Done in https://github.com/CleverRaven/Cataclysm-DDA/pull/61122/commits/5190a74c6d3833f4e3c8e245c99e97af3ff92bb8 if you'd like to review